Sample/sdk-ios-swift/ViewController.swift

```diff
@@ -10,23 +10,38 @@ class ViewController: UIViewController {
     var responseObject: ProfileDictionary?
     @IBOutlet weak var rememberMeButton: YotiButton!
     @IBOutlet weak var selfieAuthButton: YotiButton!
+    @IBOutlet weak var unfulfilledButton: YotiButton!
 
-    @IBAction func yotiButtonDidTouchUpInside(_ sender: YotiButton) {
+    func yotiButtonDidTouchUpInside(_ sender: YotiButton) {
         guard let useCaseID = sender.useCaseID else {
              return
         }
         do {
             UIApplication.shared.isNetworkActivityIndicatorVisible = true
-            try YotiSDK.startScenario(for: useCaseID, with: self)
+            try YotiSDK.startScenario(for: useCaseID, theme: sender.theme, with: self)
```

Comment:
Why do we need the theme to start the scenario?

Reply:
It slightly changes the behaviour after that. We use the theme to give some more app-specific details in the errors and in the case of EasyID always use the scheme to open the app.